packages/engine/src/scene/components/ModelComponent.ts

```diff
@@ -170,6 +169,7 @@ function ModelReactor() {
             (loadedAsset) => {
               loadedAsset.scene.animations = loadedAsset.animations
               if (!entityExists(entity)) return
+              removeError(entity, ModelComponent, 'INVALID_SOURCE')
```

Comment:
I'm still seeing lingering error messages and error component after changing the source path to an invalid extension/url and then back to the correct path. 
![image](https://github.com/EtherealEngine/etherealengine/assets/102248647/88ff524e-94a9-4aa4-ae40-8176129dd3aa)
This is probably because of the above return condition preventing the asset load logic below in this case. We should probably attempt to remove the errors before returning there.

Reply:
[abc559fb-1717-4ad1-bd15-b8055260da8a.webm](https://github.com/EtherealEngine/etherealengine/assets/55396651/6bc1871d-a609-4105-9cb3-cdb0f32675a2)

Fixed this!